infrastructure-playbooks/docker2podman.yml

```diff
@@ -0,0 +1,204 @@
+---
```

Comment:
note sure whether having a dedicated playbook is really useful in the end

I thought I could have a section like following
```yaml
- hosts: all
  tasks:
    - import_role:
        name: ceph-defaults
      tags: ['docker2podman']
    - import_role:
        name: ceph-facts
      tags: ['docker2podman']
```

so I could run those roles on all nodes at a time instead of playing them for each hosts section but I must have been confused at some point